ams/dashboard/src/components/tables-sub-menu/TablesMenu.vue

```diff
@@ -283,10 +305,14 @@ export default defineComponent({
           label: ele.name,
           type: ele.type
         }))
+        if (state.tableSearchInput === '') {
```

Comment:
I suggest moving the filtering of databases and tables from the backend to the frontend and the HTTP API always returns all databases and all tables. Then there will be no need for additional checks on whether `allTableListLoaded` should be set.

BTW, removing the filtering from the HTTP API won't incur additional overhead on the backend, because the filtering takes effect at the final step of returning results.

Reply:
Yes, filtering of databases and tables is acted in the frontend now. We only need load all databases or tables for the first time. Afterwards, keyword filtering will not request HTTP API.